src/internal/node_api_internal.h

```diff
@@ -58,8 +58,8 @@ int napi_module_init_pending(jerry_value_t* exports);
 /** MARK: - node_api_env.c */
 napi_env iotjs_get_current_napi_env();
```

Comment:
A comment not on this PR.
Why the napi implementation depends on iot.js? Theoretically, only module loading is depended on iotjs, and async callling depends on libuv.

Reply:
The naming is used to separate N-API apis from shadow-node internal apis. Or shall we name it with prefix 'shadownode' or its abbreviation?